.github/workflows/scripts/create-release.sh

```diff
@@ -23,7 +23,8 @@ create_release_notes() {
       -H "Accept: application/vnd.github.v3.raw" \
       "/repos/${GITHUB_REPOSITORY}/contents/CHANGELOG.md?ref=${TAG}"
     )"
-    ESCAPED_VERSION="${TAG//./\.}"
+    VERSION_WITHOUT_RC=${TAG/-rc.[0-9]*/}
```

Comment:
```suggestion
    VERSION_WITHOUT_RC="${TAG/-rc.[0-9]*/}"
```

Reply:
Thanks, I really need a shellcheck plugin 🙈

Reply:
After few months you can upgrade your brain to quote everything by default :)